analyzer/src/test/kotlin/managers/utils/NodeSupportTest.kt

```diff
@@ -270,7 +276,7 @@ class NodeSupportTest : WordSpec() {
         if (hasYarnLockFile) projectDir.resolve("yarn.lock").createNewFile()
     }
 
-    private fun absolutePaths(vararg files: String) = files.map { tempDir.resolve(it) }
+    private fun absolutePaths(vararg files: String): Collection<File> = files.map { tempDir.resolve(it) }
```

Comment:
Is this actually required or just "nice to have"? Should it maybe to into the `NodeSupportTest: Simplify 'absolutePaths'` commit instead?

Reply:
See below.